### PR TITLE
Color cursor visible in conversation list search

### LINKED ISSUE #5382: Text cursor invisible in conversation list search (light theme only)
### Bug description

No text cursor is visible when searching through the conversation list.
### Steps to reproduce
- Go to conversation list (light theme)
- tap the magnifier icon
- (optional) type something with the keyboard

**Actual result:** No cursor visible in input field.
**Expected result:** Blinking cursor is visible in input field.
### Device info

**Device:** Fairphone 2
**Android version:** 5.1.1
**Signal version:** 3.14.2
### Assignment

I am working on this issue.

### PR DESCRIPTION
**Pull request by FeuRenard**
### First time contributor checklist
- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist
- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Sony Xperia U, Android 4.4.4 (CyanogenMod)
 * AVD Nexus 5X, Android 6.0
 * EDIT, 4 Sep 2016: AVD Nexus 5X, Android 5.1
- [x] My contribution is fully baked and ready to be merged as is
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in my commit message

----------

### Description
Fixes #5382
Replaces #5411

// FREEBIE

### CODE REVIEW

File: res/values/themes.xml [hunk 1]
```diff
@@ -90,6 +90,7 @@
         <item name="colorAccent">@color/textsecure_primary_dark</item>
         <item name="colorControlActivated">@color/signal_primary</item>
         <item name="colorControlHighlight">@color/signal_primary</item>
+        <item name="autoCompleteTextViewStyle">@style/actionbar_cursor</item>
```
Comment (moxie0) [inline res/values/themes.xml]: What is this for?

Comment (FeuRenard) [inline res/values/themes.xml]: It's the only way I found to access the search field of the conversation list search for styling purposes. You have to consider, that there is nothing like `res/layout/conversation_list.xml` in this project.

Comment (moxie0) [inline res/values/themes.xml]: I can't really merge things that are global style changes. They would need to be specific to the individual views you're looking to restyle.

**Comment by FeuRenard** [comment]
I close this for now. I am going to reopen this PR or open a new one as soon as I'll find a more local style change achieving the same effect.

**Comment by FeuRenard** [comment]
Another way of coloring this cursor is setting `android:colorControlActivated`. But that really affects other UI elements (toggles in settings, see also #5388) and works only for API >= 21.

I don't see the proposed change affecting any other element of this app although I edit something in the global theme. Feel free to close this PR again, when you don't like this solution.

**Pull request closed without merging**